README.md

```diff
@@ -230,6 +230,8 @@ These are currently used to compute properties of a minimum energy conformation
 |`OpenFF NAGL2 ESP Timing Benchmark v1.1` | [2024-09-18-OpenFF-NAGL2-ESP-Timing-Benchmark-v1.1](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-09-18-OpenFF-NAGL2-ESP-Timing-Benchmark-v1.1) | Single point ESP calculations for timing/memory benchmarking | 'P', 'S', 'N', 'C', 'Cl', 'F', 'Br', 'O', 'H', 'I' | |
 |`OpenFF Sulfur Hessian Training Coverage Supplement v1.0` | [2024-09-18-OpenFF-Sulfur-Hessian-Training-Coverage-Supplement-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-09-18-OpenFF-Sulfur-Hessian-Training-Coverage-Supplement-v1.0) | Additional Hessian training data for Sage sulfur and phosphorus parameters (from 'OpenFF Sulfur Optimization Training Coverage Supplement v1.0') | O, S, C, Cl, P, N, F, Br, H | |
 | `OpenFF Aniline Para Hessian v1.0` | [2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-07-OpenFF-Aniline-Para-Hessian-v1.0) | Hessian single points for the final molecules in the `OpenFF Aniline Para Opt v1.0` dataset | 'O', 'Cl', 'S', 'Br', 'H', 'F', 'N', 'C' ||
+|`OpenFF Gen2 Hessian Dataset Protomers v1.0` | [2024-10-07-OpenFF-Gen2-Hessian-Dataset-Protomers-v1.0](https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2024-10-07-OpenFF-Gen2-Hessian-Dataset-Protomers-v1.0/) | Hessian single points for the final molecules in the `OpenFF Gen2 Optimization Dataset Protomers v1.0` dataset | 'H', 'C', 'Cl', 'P', 'F', 'Br', 'O', 'N', 'S'||
```

Comment:
I thought about this after Lily's review last night, but we could also link to the optimization dataset instead of just including its name. This is not blocking, just an idea.